### PR TITLE
Pod ready latency threshold

### PR DESCRIPTION
**Pull request by rsevilla87**
Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### Description

Starting up to automatically detect workload regressions from benchmarks.

Configuring a pod ready latency threshold of 5000ms (by default) for node/pod-density workloads, hence, kube-burner will exit 1 when pod ready latency is above the configured value. Source [Upstream SLO](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md)

The error will be displayed in the kube-burner log, so it will be easily recognizable as soon as https://github.com/cloud-bulldozer/e2e-benchmarking/pull/327 lands

### CODE REVIEW

**Review by mukrishn — APPROVED** [review]
LGTM

**Review by smalleni — COMMENTED** [review]
Shouldn't POD_READY_THRESHOLD be 5000ms?

File: ci/env.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ export CLEANUP_WHEN_FINISH=true
 export NODE_COUNT=1
 export PODS_PER_NODE=75
 export PODS=50
+export POD_READY_THRESHOLD=1m
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply ci/env.sh:19]: This is just the CI value :)

File: workloads/kube-burner/env.sh [hunk 2]
```diff
@@ -44,3 +44,6 @@ export LOG_LEVEL=${LOG_LEVEL:-info}
 # Pprof
 export PPROF_COLLECTION=${PPROF_COLLECTION:-false}
 export PPROF_COLLECTION_INTERVAL=${PPROF_COLLECTION_INTERVAL:-5m}
+
+# Thresholds
+export POD_READY_THRESHOLD=${POD_READY_THRESHOLD:-5000ms}
```
Comment (rsevilla87) [inline workloads/kube-burner/env.sh:49]: cc: @smalleni

Comment (smalleni) [reply workloads/kube-burner/env.sh:49]: gotcha.

**Review by smalleni — APPROVED** [review]

**Review by mohit-sheth — APPROVED** [review]

**Pull request merged**